src/main/java/com/groupseven/sysc4806project/Book.java

```diff
@@ -44,14 +54,56 @@ public void setPublisher(String publisher) {
         this.publisher = publisher;
     }
 
-
     @Id
     @GeneratedValue
     public int getId() {
         return id;
     }
 
-    public String image() {return "";}
+    @Transient
+    public String getImageUrl() {
+        return "/" + id + ".png";
+    }
+
+    @Transient
```

Comment:
What is the tag transient for?

Reply:
Transient tells Spring to ignore this field when saving/loading it in a database. Just marking this method as a way to generate data rather than saving it. Similarly, `@JsonIgnore` tells Spring to ignore this field when returning an object as JSON. We don't want to call `setImage()` when creating an object, so it is ignored.

Reply:
Alright, thanks for the explanation